app-emulation/wineasio/wineasio-9999.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2010 Gentoo Foundation
```

Comment:
Update the the header and "sign" the contribution like other ebuilds in the repository.